data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/threshold/AbstractThreshold.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_impl.remedial_action.threshold;
```

Comment:
It might more consistent to move threshold package out of remedial_action package because thresholds can be related to Cnecs as well.